docs/Parameters.rst

```diff
@@ -336,6 +338,28 @@ Learning Control Parameters
 
    -  **Note**: if balanced bagging is enabled, ``bagging_fraction`` will be ignored
 
+-  ``mvs_lambda`` :raw-html:`<a id="mvs_lambda" title="Permalink to this parameter" href="#mvs_lambda">&#x1F517;&#xFE0E;</a>`, default = ``1e-4``, type = double, constraints: ``mvs_lambda > 0.0``
+
+   -  used in MVS boosting if ``mvs_adaptive == true`` than this value is ignored
```

Comment:
```suggestion
   -  used in MVS boosting. If ``mvs_adaptive == true`` than this value is ignored
```

Reply:
BTW, `than` should be `then`, could your please fix this together. Sorry for missing the typo.